inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/exporter/KnowledgeBaseExporter.java

```diff
@@ -224,14 +224,9 @@ public void importData(ProjectImportRequest aRequest, Project aProject,
             kb.setReification(Reification.valueOf(exportedKB.getReification()));
             kb.setBasePrefix(exportedKB.getBasePrefix());
 
-            if (exportedKB.getRootConcepts() != null) {
-                kb.setRootConcepts(
-                    exportedKB.getRootConcepts().stream()
-                        .map(conceptId -> vf.createIRI(conceptId)).collect(Collectors.toList()));
-            }
-            else {
-                kb.setRootConcepts(new ArrayList<>());
-            }
+            kb.setRootConcepts(exportedKB.getRootConcepts() != null ? exportedKB.getRootConcepts().stream()
```

Comment:
Not sure if this is cognitively easier to process...

Reply:
The changes here are a bit too deliberate. I originally wanted to reduce an if loop statement, but it actually has no effect......